types/module/module.go

```diff
@@ -367,11 +367,6 @@ func (m *Manager) InitGenesis(ctx sdk.Context, cdc codec.JSONCodec, genesisData
 		}
 	}
 
-	// a chain must initialize with a non-empty validator set
```

Comment:
This was added in v0.46 (so it is normal it isn't on v0.45.x and v0.45.x-ics). Could you please tell us which issues you are facing with it?

Reply:
I cannot, but @vuong177 can.  He is the person who identified this stuff.